app/graphql/client.tsx

```diff
@@ -95,12 +95,24 @@ const GaloyClient: React.FC<PropsWithChildren> = ({ children }) => {
         }`,
       )
 
+      const wsLinkAuth = async () => {
```

Comment:
oh so we need to do a new rollout before activating DeviceAccount?

Reply:
This is no longer relevant with required changes to device account authentication